test/standard_test.js

```diff
@@ -1,20 +0,0 @@
-var standard = require('mocha-standard')
```

Comment:
The thing is that declaring standard dep we can do the same just declaring lint command before npm-test. Also Atom detect that standard is declared and auto-format the code